src/components/ActiveNoteDisplay.js

```diff
@@ -0,0 +1,16 @@
+import { useStyles } from "../styles/styles";
+import Typography from "@material-ui/core/Typography";
+
+const ActiveNoteDisplay = (props) => {
```

Comment:
No need to set Display inside name, `ActiveNote` will be enough.
Destructure please props and set PropTypes